content/browser/media/capture/desktop_capture_device_mac.cc

```diff
@@ -6,7 +6,6 @@
 
```

Comment:
Similarly to what I mention below, since we don't support desktop
capture on Cobalt, we could just not build this file by setting
`enable_screen_capture` to false in args.gn and that'd make
modifications to this file unnecessary (and perhaps save some
bianry size).

[enable_screen_capture](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:content/common/features.gni;l=23-24?q=enable_screen_capture&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2Fmain:content%2F)